bin/webindex

```diff
@@ -149,16 +149,6 @@ load-s3)
     ${COMMAND}
   fi
   ;;
-reindex)
```

Comment:
While I think it's OK that the reindex command is removed for this PR, I think it is something we should tackle.  I think it's a problem that all production Fluo applications will need to solve and we should work through it to determine if new features or recipes are needed and show that is possible.  What do you think about adding this as an issue?